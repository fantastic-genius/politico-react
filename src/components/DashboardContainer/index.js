import React from 'react';
import DashboardNavBar from '@components/DashboardNavBar';
import UserDashboardSideBar from '@components/UserDashboarSideBar';
import Footer from '@components/Footer';

const DashboardContainer = ({ children }) => {
  return (
    <div className="container dashboard">
      <DashboardNavBar />
      <div className="main">
        <UserDashboardSideBar />
        {children}
      </div>
      <Footer />
    </div>
  );
}

export default DashboardContainer;
