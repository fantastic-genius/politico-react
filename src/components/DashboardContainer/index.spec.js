import React from 'react';
import { shallow } from 'enzyme';
import DashboardContainer from './index';

describe('<DashboardContainer />', () => {
  describe('Rendering', () => {
    it('should render DashboardContainer correctly in debug mode', () => {
        const component = shallow(<DashboardContainer />);
        expect(component).toMatchSnapshot();
    });
  })

})

