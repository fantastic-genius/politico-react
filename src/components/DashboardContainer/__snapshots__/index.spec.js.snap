// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<DashboardContainer /> Rendering should render DashboardContainer correctly in debug mode 1`] = `
<div
  className="container dashboard"
>
  <DashboardNavBar />
  <div
    className="main"
  >
    <UserDashboardSideBar />
  </div>
  <Footer />
</div>
`;
